proto/cosmos/tx/v1beta1/service.proto

```diff
@@ -170,4 +188,32 @@ message GetBlockWithTxsResponse {
   .tendermint.types.Block       block    = 3;
   // pagination defines a pagination for the response.
   cosmos.base.query.v1beta1.PageResponse pagination = 4;
-}
\ No newline at end of file
+}
+
+// TxDecodeRequest is the request type for the Service.TxDecode
+// RPC method.
+message TxDecodeRequest {
```

Comment:
let's also add since comments for these 4 structs

Reply:
added, but `make proto-gen` is not working on my system for some reason idk.